css_tasks/task5/main.css

```diff
@@ -0,0 +1,283 @@
+:root {
+  --fontColor: rgb(18, 18, 18);
+  --mainThemeColor: red;
+  --secondaryThemeColor: blue;
+  --headerFont: "Cinzel", serif;
+  --mainThemeFont: "Ubuntu", sans-serif;
+  --secondaryThemeFont: "Roboto", sans-serif;
+  --articleHeaderFont: "Bitter", sans-serif;
+  --figcaptionColor: grey;
+}
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  color: var(--fontColor);
+}
+.container {
+  width: 80%;
+  margin: -1rem auto 0;
+}
+/**********************
+*                     *
+*     TOGGLERS        *
+*                     *
+***********************/
+.container__toggler {
+  position: fixed;
+  margin-left: 20px;
+  left: 0;
+  cursor: pointer;
+  padding: 5px;
+  border: 1px solid var(--fontColor);
+  border-radius: 5px;
+  width:100px;
+  font-family: var(--mainThemeFont);
+  font-size:.77rem;
+  text-align:center
+}
+.toggler--theme {
+  top: 48%;
+}
+.toggler--font {
+  top: 52%;
+}
+.toggler--font-size{
+    top:56%;
+}
+
+.theme-toggler,
+.font-toggler,
+.font-size-toggler {
+  -webkit-appearance: none;
+}
+/**********************
+*                     *
+*      HEADER         *
+*                     *
+***********************/
+.header {
+  width: 100%;
+  height: 7rem;
```

Comment:
generally it is better not to specify height of elements, let your content to do this(or sometimes do it by paddings), you can remove it with margin-top on content__main-section and everything will looks fine